GeosCore/tomas_mod.F90

```diff
@@ -6051,7 +6148,7 @@ SUBROUTINE AERODIAG( PTYPE, I, J, L, Nk, Nkd, Mk, Mkd, BOXVOL, State_Grid )
 !
 ! !LOCAL VARIABLES:
 !
-    INTEGER               :: K, JS
+    INTEGER               :: K, JS, S
     REAL*4                :: ADXX(IBINS*(ICOMP-IDIAG+1))
     REAL*4,    SAVE       :: ACCUN, ACCUM(2)
     LOGICAL,   SAVE       :: FIRST = .TRUE.
```

Comment:
These variables can be removed. They were for BPCH diagnostics code that has been commented out/removed below.

Reply:
Now resolved in commit [c4a3708](https://github.com/geoschem/geos-chem/pull/1378/commits/c4a37080b06dbe9c228fffce228e231ae010ccbc).